src/discord_utils.py

```diff
@@ -93,6 +97,15 @@ async def scrape_channel_media(
                 )
             )
 
+    # Clear unused files in attachment directory
```

Comment:
```suggestion
    # Clear unused files in this guild's attachment directory
```

Just so it doesn't sound like we're deleting any attachments related to other guilds.